.gitignore

```diff
@@ -2,4 +2,7 @@
 .cache
 dist
 node_modules
-*.sw*
\ No newline at end of file
+*.sw*
+.idea
+.env
+.parcel-cache
```

Comment:
Should we add the .vscode folder as well?

Reply:
I would say yes